app/modules/import-export/module.php

```diff
@@ -86,6 +86,8 @@ public function __construct() {
 		if ( ElementorUtils::is_wp_cli() ) {
 			\WP_CLI::add_command( 'elementor kit', WP_CLI::class );
 		}
+
+		( new Usage() )->register();
```

Comment:
Not critical at all, but maybe rename "register()" to something more self-explanatory?

Reply:
Any suggestions?

Reply:
send_tracking?
send_tracking_data?
push_usage_data?
set_usage_data?

Reply:
Done

Reply:
I think `register` is fine. It's the method that registers the service actions (same as we did in the Notes)
See Laravel's `register` and `boot` methods for reference:
https://laravel.com/docs/9.x/providers#the-register-method
https://laravel.com/docs/9.x/providers#the-boot-method